components/Layout.tsx

```diff
@@ -369,6 +369,9 @@ const Footer = () => (
         </div>
       </div>
     </div>
+    <div className='copyright-section'>
+      <p>Copyright © 2024 JSON Schema. All rights reserved.</p>
```

Comment:
Why this change is part of the PR?

Reply:
He should learn git first!

Reply:
Are you mentor of this organization and open source contribution is about collaborating and learning. :)